src/xkbcomp/parser.y

```diff
@@ -4,9 +4,7 @@
  */
 
 /*
- * The parser should work with reasonably recent versions of either
- * bison or byacc.  So if you make changes, try to make sure it works
- * in both!
+ * The parser should work with reasonably recent versions of bison.
```

Comment:
It would be great to have a version range here. But since it is the role of the meson file, maybe remove the comment entirely, so we only handle the version in one place.

Maybe instead we could document here why we drop `byacc`:

```suggestion
 * The parser supports only bison, because byacc does not support the following features: …
```